package.json

```diff
@@ -57,19 +56,20 @@
     "http-server": "^0.11.1",
     "merge2": "^1.2.1",
     "nedb": "^1.8.0",
-    "nyc": "^11.7.1",
+    "nyc": "^11.9.0",
     "pre-commit": "^1.2.2",
     "tap-bark": "^1.0.0",
-    "ts-node": "^6.0.2",
-    "tslint": "^5.9.1",
+    "ts-node": "^6.2.0",
+    "tslint": "^5.11.0",
     "typedoc": "^0.11.1"
   },
   "dependencies": {
+    "@types/node": "^10.9.4",
```

Comment:
Any reason the this has been added to the dependencies?

Reply:
Yes, was needed to allow the internal module imports, but I found another solution for that. So I'll remove it.